05_plot_cancer_type_results.ipynb

```diff
@@ -0,0 +1,1539 @@
zz//vN9x99xzDytWrGDFihX07dvXd8/jxo1jzpw5gFfOF198MZMmTeLqq69m1KhRfP75574xFixY4Dtv7NixzJ8/37cvKyuLF1544ajzPJFKcgsWLGDUqFG89957vPjii4wePZoVK1Zwzz33HNf5GRkZTJo0CTg5mUokf4ZGUbFNIvkr+eabbxg5ciTffffdcRXnuO6663w1BzIyMpg6dSpffPEFALNmzfLVE1+8eDEzZszgjTfe4O2336a4uJj3338fRVFIS0vj9ttvZ/78+bjdbr7++mtmzpxZ5/Xat29P+/btT9HdHuL6669n6tSpvPnmm8d9zty5c7n++uv58MMP6dy583Gd06dPH1588UUAXC4XI0aM4NJLLwVg2rRpviJMJSUlvgIp69atY/bs2bzxxhvY7XaKi4sZP348bdq0oU2bNjzzzDM88cQTJ3jHdfPrr78yZcoUhgwZwvDhw/niiy/YvHnzSY11MjKVSP4MUolLzmpWrFhBYmIiV155JdOmTTvhClslJSUEBQXVuS8nJ8dX7vaTTz5h3rx5vradnTt35rPPPsNisTBv3jyf5ZiZmck///lPNE1DVVWeffZZsrKy+Pjjj3nxxReZO3cuH3zwAWFhYVgsFkaOHAnA77//jsPhIDs7m5tvvpnRo0ezcuVKnwXvcDh45plnsFgsvvmFhoZis9nYtm0bKSkpx7zXyspKli9fznfffcfFF19MUVFRne1/66OiogJFUeqsEFhQUIDVakUIwdy5c7n22mux2+0AREREMHfuXEJDQ9m1axemaRIZGYnT6eSuu+6ioqLC1363d+/euFwu7r33XnJycggPD2f69Om8/vrrvqJPGRkZPPLII1x33XX89ttvpKWlsWXLFvbv38+tt97KLbfc4pvXDz/8wOzZs1EUhR49ejB16lQOHDjA1KlTfS01T1amEsmfRSpxyVnN3LlzGTt2LElJSVitVjZs2FDncYfXCJ89ezbff/89iqIQGhrKY4895tt3ww034HQ6OXDgAAMGDOD+++8HvEo0LCzMb8yIiAgAVq5c6Xt5WLZsGR07duSBBx5g9erVlJaW+o4vKiri7bff5ssvv8RqtXLNNdf49lVUVDBz5kyysrKYPHkyo0ePZseOHTz33HPExsby+uuvM3/+fC6++GK/ObRr146VK1cel8L5/vvvGT58ODabjQsvvJDPPvvMT9nVJTPTNFm+fDmTJk1CCIHFYuGhhx7yKefnnnuO119/nZycHFq3bs1LL70EwIEDB0hISPAb76D8Vq1aRbt27QDIzs6moKCA2bNnU1hYSFZWFgBVVVXcc889xMfHM2nSJLZu3VrnHIcOHcrPP//MyJEjGThwIPPmzWPWrFmsX78e8L6kvfzyy3z++ecEBgYybdo0li5dypIlSxg1ahTjxo3j+++/56OPPjopmUokfxapxCVnLaWlpSxatIiioiLmzJlDRUUF77//PkFBQbhcLr9jPR6P7/+Hu9OP5KA7/X//+x979+4lKioK8FpoR3Yw+vnnn+nbty/FxcW+46644greeustbrrpJkJCQvzWZbOzs2ndurWvZvnhJXIPKoy4uDjf3GNjY3niiScICgoiLy+P7t2715pvTEwMeXl5vr8rKyuZPHkyAP369eO2227z7Zs7dy6qqnLjjTficDjYv38/N910EwEBAbXkVVVVRUBAANXV1X7u9CM56E7//fffef7550lMTASgWbNm5Obm+inCNWvWEB0d7Sev5ORkJk6cyJQpU/B4PL616bCwMOLj4wGIjo6murq6zusfi+zsbIqKinwvK5WVlezZs4cdO3b4lgS6d+/up8SPlKlE8lciA9skZy1ff/01Y8aMYdasWcycOZNPP/2UpUuXkpCQ4Bdktnr1al/npePl7rvv5sCBA3z44YcAXH755cyYMYODVY7Xrl3LU089hdVqJTIy0tcqduHChfTo0YN3332XESNG8Pbbb/vGTExMZNeuXTgcDgzD8Asuq6ub2IMPPsiTTz7J008/TZMmTairwnJpaalPIYK3/eqcOXOYM2eOnwJPT09H13U++ugjZs6cyQcffEBiYiK//vorKSkpLFu2zNedqaSkhB07dtC6devjltd5553H0KFDeeihhwAYPXo0M2fOpKqqCoDCwkL+9a9/UV1dTVRUlK8xRHp6OpWVlbz55ps8/fTTPq9IXfKw2Wzk5+cDHPead3x8PHFxccyaNYs5c+Zw9dVX06VLF5KSkli3bh2AL3jxIEfKVCL5K5GWuOSsZe7cuTz77LO+vwMDAzn//PNxOBwEBQVx6aWXYrfbsVgsPProoyc0tqIoPPHEE0ycOJFhw4Zx44038tJLLzF+/Hg0TUPTNF577TWsViu9e/dmw4YN9OrVi9TUVKZNm8bLL7+Moij885//9PU0j4yM5Oabb2bChAmEh4fjdDrRNM3PS3A4l156KePGjSM0NJTo6GgOHDhQ65i0tLTjisKeO3euz/I8yNixY/nggw+YNWsWEyZMYMKECdjtdjweD//+9799LvPj5fbbb2f06NH89ttvDBo0iHHjxnHDDTegaRoOh4MpU6aQkpJCYGCgL6itZcuWvPLKK3z55ZdYLBbuvPPOo45/4YUXcvfdd7Nq1SpSU1OPa06RkZFcd911TJo0CV3Xad68ORdeeCF33XUX99xzD99//73P4j/I8cpUIjklmBKJpEGpqKgwr7nmmmMe53a7zVdffdX394QJE8yVK1ee9HWLi4vNW2+99aTPb0huvfVWMz8/v6GnUYvGLFNJ40S60yWSBsZut3PZZZfx448/1nucpmlUV1dz+eWXM27cONq3b0/Pnj1P+rqzZ89utBbjtGnTeOeddxp6GrVozDKVNE5kK1KJRCKRSBop0hKXSCQSiaSRIpW4RCKRSCSNFKnEJRKJRCJppEglLpFIJBJJI0UqcYlEIpFIGilSiUskEolE0kj5/5XlCMPCBy0bAAAAAElFTkSuQmCC\n",
```

Comment:
I am a little lost on what these plots are telling you, but I think that is because I got a little lost on what these different df are from above.

Reply:
Looking back now, these plots make a lot more sense now. For your last plot where you're identifying cancer genes that were better predicted using your pan-cancer model vs your single cancer model, were you able to identify the `certain effects (e.g. in relatively rare cancer types) ` you mentioned? I guess I'm not sure what you mean by these effects? Do you mean genes that are rarely mutated except in a single cancer type that has a low sample count?

Reply:
>  were you able to identify the `certain effects (e.g. in relatively rare cancer types)` you mentioned?

Ehh, kinda. I was hoping to identify more examples that clearly benefit from adding the pan-cancer data (in the top right of the last plot), but there aren't too many unfortunately.

I have some ideas for how to interpret the last result, but I haven't talked with Casey about this plot yet, so he might have ideas on what it means.

> I guess I'm not sure what you mean by these effects? Do you mean genes that are rarely mutated except in a single cancer type that has a low sample count?

I was anticipating that for some genes, there would be a strong signal in one cancer type, but this signal would get washed out when the pancancer data was added (i.e. the gene has strong cancer-specific effects that aren't preserved in other cancer types). We don't really see much of that, though, which in retrospect isn't that surprising since we have a model covariate for cancer type (so the model can just choose to put lots of weight on samples from the relevant cancer type). I should **definitely** check if this is actually the case, which I think would be a neat follow-up (#23).

I was also anticipating that for some genes, there wouldn't be a strong signal in one cancer type, but the signal would be preserved across cancer types so adding more data would lead to a better model. This somewhat seems to be the case for a few genes (in particular, NF1 and FBXW7 seem to show up frequently) but I need to think more about what this means and why these genes.